thirdeye/thirdeye-frontend/app/pods/components/alert-report-modal/component.js

```diff
@@ -82,7 +82,7 @@ export default Component.extend({
    * @method bubbleModalInput
    */
   bubbleModalInput() {
-    this.sendAction('inputAction', this.get('reportAnomalyPayload'));
+    this.get('inputAction')(this.get('reportAnomalyPayload'));
```

Comment:
You can do this.inputActions(...) if you want.

Reply:
I personally would keep it `get` as that's more proper way of getting and setting props for Ember.Object.

Reply:
That is changing ;)
https://github.com/emberjs/rfcs/blob/master/text/0281-es5-getters.md

Reply:
Man. Ember keeps changing like this. Soon we just do regular vanilla. :)